chain/beacon/store.go

```diff
@@ -129,6 +131,14 @@ func (d *discrepancyStore) Put(b *chain.Beacon) error {
 	return nil
 }
 
+func (d *discrepancyStore) ForcePut(b *chain.Beacon) error {
+	if err := d.Store.ForcePut(b); err != nil {
+		return err
+	}
+	d.l.Infow("ForcePut", "NEW_BEACON_STORED", b.String())
```

Comment:
do we want to re-use this `new_beacon_stored` string?
consider using a unique identifier here, like `HISTORIC_BEACON_STORED`